src/main/java/ai/elimu/web/content/sound/SoundEditController.java

```diff
@@ -52,7 +52,7 @@ public String handleRequest(Model model, @PathVariable Long id) {
         
         model.addAttribute("soundContributionEvents", soundContributionEventDao.readAll(sound));
         
-        model.addAttribute("letterSoundCorrespondences", letterSoundDao.readAll());
+        model.addAttribute("letterSounds", letterSoundDao.readAll());
```

Comment:
@Sneha65 At http://localhost:8080/webapp/content/sound/edit/1, I cannot see the list of letter-sounds anymore:

<kbd>![](https://github.com/user-attachments/assets/1c71fe13-aafb-49f1-bb99-eca1e8e96756)</kbd>

So you would probably have to update the attribute name in `src/main/webapp/WEB-INF/jsp/content/sound/edit.jsp` as well.